ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-adaptive-virtual-topology.j2

```diff
@@ -0,0 +1,35 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - router-adaptive-virtual-topology #}
+{% if router_adaptive_virtual_topology is arista.avd.defined %}
+
+### Router Adaptive Virtual Topology
+
+#### Router Adaptive Virtual Topology Summary
+
+| Settings | Value |
+| -------- | ----- |
+{%     if router_adaptive_virtual_topology.topology_role is arista.avd.defined %}
+| Topology Role | {{ router_adaptive_virtual_topology.topology_role }} |
+{%     endif %}
+{%     if router_adaptive_virtual_topology.region is arista.avd.defined %}
+| Region | {{ router_adaptive_virtual_topology.region.name }} id {{ router_adaptive_virtual_topology.region.id }} |
```

Comment:
```suggestion
| Region Name | {{ router_adaptive_virtual_topology.region.name }} |
| Region ID | {{ router_adaptive_virtual_topology.region.id }} |
```

Reply:
resolved in [7748d7d](https://github.com/aristanetworks/ansible-avd/pull/3237/commits/7748d7d4953ff9c03a7feb548cb05d7dadad5af5)